docs/platform/how-to/deploy-public-page.md

```diff
@@ -0,0 +1,49 @@
+# Public services
+
+The Quix SaaS platform allows you to deploy public facing web pages and API's.
+
+## Library samples
+
+In our Library you can find our `Web API Template` which demonstrates how to create a API using `Node`.
+
+There are also examples dashboard and web/UI examples using `Dash`, `Streamlit` and `Angular`.
+
+![Library examples](../images/library.png){width=500px}
+
+## The code
+
+When running locally it is usual to see local addresses like `127.0.0.1` and `localhost`, however, these cannot be used and will result in routing errors.
+
+Please ensure your service is hosted on `0.0.0.0` and uses port `80`.
+
+Examples of this for various languages are:
+
+ - Node JS which can be used to host Dash apps and any Javascript based websites:
```

Comment:
"Node JS" -> Node.js
Javascript -> JavaScript.
JavaScript-based.

What are Dash apps? Do we need to provide a link?